virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,18 +1,26 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to an external network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal `pod` network. You update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration] depending on the type of network to which you want to connect.
```

Comment:
The second sentence is the main subject of this section but it gets lost in this context. Also, it's too vague.

Recommend this:

"By default, {VirtProductName} is installed with a single, internal `pod` network. 

You must create a Linux bridge network attachment definition in order to connect to external networks."

I think we need to phrase it this strongly because connecting to external networks is pretty much a requirement for most users, not an option, unless they're setting up a sandbox or disconnected environment. Since the title makes it obvious that it's about how to configure CNV for external networks, the Linux bridge is a requirement. So "depending on the type of network" becomes meaningless in this context.

Reply:
Here's the latest version of the intro:

By default, {VirtProductName} is installed with a single, internal pod network.

You must create a Linux bridge network attachment definition in order to connect to external networks. Otherwise, your network interface cards will be unavailable.

Once a Linux bridge network attachment is created, you can start virtual machines on the external network in the same manner that you start virtual machines attached to internal networks.

To attach a virtual machine to an external network:

. Configure a Linux bridge.
. Edit the virtual machine configuration to attach it to the bridge network attachment definition.
+
[NOTE]
====
The network attachment definition must be in the same namespace as the pod or virtual machine.
====
. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
** In the web console, create a NIC for a new or existing virtual machine.
** In the CLI, include the network information in the virtual machine configuration.

[NOTE]
====
There are multiple methods for configuring a VLAN, for example, creating a node network configuration policy. However, a network attachment definition provides a more efficient and manageable configuration
====